app/models/git/concept.rb

```diff
@@ -22,6 +21,11 @@ def about
       read_file_blob('about.md')
     end
 
+    memoize
+    def introduction
+      read_file_blob('introduction.md')
+    end
```

Comment:
While not strictly belonging to this PR, I've snuck this in to allow retrieving the `introduction.md` document.

Reply:
We can write a nice DSL to replace these once this PR is merged :)